src/lev-report/index.js

```diff
@@ -48,6 +49,7 @@ class LevReport extends React.Component {
   }
 
   render() {
+    console.log(this.state);
```

Comment:
This looks like a debugging statement that should be removed. (Perhaps it is in subsequent commits; I've not go there yet.)

Reply:
-

Reply:
Removed